workflows/cluster_plots/__main__.py

```diff
@@ -0,0 +1,47 @@
+import os
+
+import luigi
+from plotting import ClusterPlottingTask
+
+from birdclef.knn_labels import (
+    compute_embedding_2d,
+    get_knn_labels,
+    get_label_agreement,
+    get_subset_pdf,
+    plot_distances,
+    plot_embedding,
+    write_plots_to_disk,
+)
+from birdclef.utils import get_spark
+
+
+class ClusterPlotTask(luigi.WrapperTask):
+    path_prefix = luigi.Parameter()
+
+    def requires(self):
+        spark = get_spark(memory="16g")
```

Comment:
I'd probably set the memory a bit lower here, the embedding dataset is only 500mb or so. Default is 2gb iirc.